packages/commonwealth/client/scripts/views/components/component_kit/CWContentPage/CWContentPage.tsx

```diff
@@ -170,13 +170,7 @@ export const CWContentPage = ({
   const mainBody = (
     <div className="main-body-container">
       <div className="header">
-        {typeof title === 'string' ? (
-          <CWText type="h3" fontWeight="semiBold">
-            {title}
-          </CWText>
-        ) : (
-          title
-        )}
+        {typeof title === 'string' ? <h1 className="title">{title}</h1> : title}
```

Comment:
it is a bit weird that we use h1 but we make it to look like h3 🤔 I know it is for SEO but this breaks our design rules system etc. Can't it be simply CWText type='h1' ?

Reply:
[CWText type="h1"](https://github.com/hicommonwealth/commonwealth/blob/master/packages/commonwealth/client/styles/components/component_kit/cw_text.scss#L30) has `font-size: 36px` and `line-height: 48px` which are different from current thread title styling

`CWText` component just applies styling to the text, not HTML tags. Even if we use `CWText type='h1'` it will just be a `div` tag.